app/controllers/organisation_controller.rb

```diff
@@ -4,6 +4,8 @@ class OrganisationController < ApplicationController
     'Crown Commercial Services'
   ]
 
+  GDS_ORG_ACCOUNT_VENDING_README = 'https://github.com/alphagov/gds-aws-organisation-accounts#requesting-a-new-aws-account-in-gds'
```

Comment:
should this link to https://engineering-enablement.gds-reliability.engineering/interim-org-platform.html instead? 

(or https://engineering-enablement.gds-reliability.engineering/aws.html if https://github.com/alphagov/ee-engineering-hub/pull/78 is merged)